projects/opendal/build.sh

```diff
@@ -25,6 +22,12 @@ targets=(
   fuzz_reader
   fuzz_writer
 )
-for target in "${targets[@]}"; do
-  cp ../target/x86_64-unknown-linux-gnu/release/$target $OUT/
-done
+
+cp ../target/x86_64-unknown-linux-gnu/release/fuzz_reader $OUT/fuzz_reader_fs
+cp ../target/x86_64-unknown-linux-gnu/release/fuzz_reader $OUT/fuzz_writer_fs
+
+cp ../target/x86_64-unknown-linux-gnu/release/fuzz_reader $OUT/fuzz_reader_memory
+cp ../target/x86_64-unknown-linux-gnu/release/fuzz_reader $OUT/fuzz_writer_memory
+
+echo -e "OPENDAL_FS_TEST=on\nOPENDAL_FS_ROOT=/tmp" > $OUT/.fs.env
```

Comment:
I prefer to create `.fs.env` by here doc:

```shell
cat <<EOF > $OUT/.fs.env
OPENDAL_FS_TEST=on
OPENDAL_FS_ROOT=/tmp
EOF
```

I find this way is easy to maintain.